oceannavigator/frontend/src/components/DatasetSelector.jsx

```diff
@@ -373,7 +391,7 @@ class DatasetSelector extends React.Component {
 
     let depthSelector = null;
     // eslint-disable-next-line max-len
-    if (this.props.showDepthSelector && this.state.datasetDepths && this.state.datasetDepths.length > 0 && !this.state.loading) { 
+    if (this.props.showDepthSelector && this.state.datasetDepths && this.state.datasetDepths.length > 0 && !this.state.loading && !this.state.two_dimensional) { 
```

Comment:
Could we make this new state variable a little more descriptive? Like `variable_two_dimensional` for example.